pkgs/development/python-modules/bacpypes/default.nix

```diff
@@ -1,15 +1,16 @@
 { lib, buildPythonPackage, fetchPypi, fetchFromGitHub
-, wheel, pytestCheckHook, pytestrunner }:
+, wheel, pytestCheckHook, pytestrunner, pythonAtLeast }:
 
 buildPythonPackage rec {
-  version = "0.18.1";
+  version = "0.18.4";
   pname = "bacpypes";
+  disabled = pythonAtLeast "3.9";
```

Comment:
https://github.com/JoelBender/bacpypes/blob/0.18.4/setup.py#L13-L26

Reply:
this looks right